Tests/BridgetTests/BridgetTests.swift

```diff
@@ -1,12 +1,12 @@
 import XCTest
-@testable import mobile_apps_thrift_swift
+@testable import Bridget
 
-final class mobile_apps_thrift_swiftTests: XCTestCase {
+final class BridgetTests: XCTestCase {
     func testExample() {
         // This is an example of a functional test case.
         // Use XCTAssert and related functions to verify your tests produce the correct
         // results.
-        XCTAssertEqual(mobile_apps_thrift_swift().text, "Hello, World!")
+        XCTAssertEqual(String(describing: Bridget.AdSlot.self), "AdSlot")
```

Comment:
The old tests didn't even compile, so I've added this so at least it does now.

Reply:
We could actually delete all the tests since this is a generated repository.

Reply:
I'm up for that, doesn't feel like this test is doing much! Maybe a separate PR?